pages/vi/faq.md

```diff
@@ -66,6 +67,20 @@
     - Name the organization '&lt;YourUserName&gt;-OLE', select 'My personal account', choose the 'Team For Open Source' option, and enter your email address to create the account.
     - Now it will prompt you for which profile to use when you fork the [treehouses repository](https://github.com/treehouses/treehouses.github.io).
 
+#### Q11: What if you encounter Host key verification fail?
+
++ To fix the issue, please type  `rm YourHomeDirectory/.ssh/known_hosts` in your powershell or terminal.
+
+For example: I would type
+
+ *In linux (ubuntu, fedora, ... )*  
+ 
+ `rm "/home/$(whoami)/.ssh/known_hosts"`
+       
+*In Windows*
+
+ `rm C:\Users\"your_user_name"\.ssh\known_hosts`
+
```

Comment:
also there are at least two more options
this one here I would call the nuclear one ;)

for more info look at https://linuxcommando.blogspot.com/2008/10/how-to-disable-ssh-host-key-checking.html